tests/test_extensions.py

```diff
@@ -46,9 +46,11 @@ def test_gaussian_sample():
 
 def test_ms_ssim():
     l = ext.loss('ms_ssim')
-    assert l(tf.zeros((1, 180, 180, 1)), tf.zeros((1, 180, 180, 1))) == 0.0
+    assert tf.math.reduce_mean(l(tf.zeros((1, 180, 180, 1)),
```

Comment:
why?

Reply:
This turned out to be caused by a weird tensorflow behavior, so the test change has been reverted.